File: Python-Programing-Basics/For Loop/For Loop - Exercise/06_salary/salary.py
tabs_in_browser = int(input())
salary = int(input())

for i in range(tabs_in_browser):
    tab_name = input()
    if tab_name == "Facebook":
        salary -= 150
    elif tab_name == "Instagram":
        salary -= 100
    elif tab_name == "Reddit":
        salary -= 50

    if salary <= 0:
        print("You have lost your salary.")
        break

if salary > 0:
    print(salary)

File: JavaScript/Programming Basics - JavaScript/For Loop/salary.js
function calculateTicketForTab(input){
    let numOfTabs = Number(input[0]);
    let salary = Number(input[1]);
    for (let i = 2; i <= input.length; i++){
        let tabName = input[i];
        switch(tabName){
            case 'Facebook': salary -= 150; break;
            case 'Instagram': salary -= 100; break;
            case 'Reddit': salary -= 50; break;
        }
        if (salary <= 0) {
            console.log("You have lost your salary.");
            return;
        }
    }
    console.log(salary);
}

calculateTicketForTab(["10",
"750",
"Facebook",
"Dev.bg",
"Instagram",
"Facebook",
"Reddit",
"Facebook",
"Facebook"]);

calculateTicketForTab(["3",
"500",
"Github.com",
"Stackoverflow.com",
"softuni.bg"]);